src/pages/api/meetings/guest/notify.ts

```diff
@@ -0,0 +1,50 @@
+import * as Sentry from '@sentry/nextjs'
```

Comment:
Why do we need a new endpoint for this ? the current sync and notify already does this no ?